src/Microsoft.AspNetCore.SystemWebAdapters.CoreServices/RemoteAppClientOptions.cs

```diff
@@ -31,8 +31,7 @@ public class RemoteAppClientOptions
     public Uri RemoteAppUrl { get; set; } = null!;
 
     /// <summary>
-    /// Gets or sets an <see cref="HttpClient"/> to use for making requests to the remote app.
-    /// A new <see cref="HttpClient"/> will be automatically generated if none is provided.
+    /// Gets or sets an <see cref="HttpMessageHandler"/> to use for making requests to the remote app.
     /// </summary>
-    public HttpClient? BackchannelHttpClient { get; set; }
+    public HttpMessageHandler? BackchannelHandler { get; set; }
```

Comment:
@mjrousos I had to change this to a handler so that it could be fit into the HttpClientFactory pipeline. It seems reasonable to me, but maybe I'm missing something?

Reply:
There was some discussion about whether to use an HttpClient or HttpMessageHandler during the API review. We landed on an HttpClient since that provides the most flexibility ([link](https://github.com/dotnet/aspnetcore/issues/42092#issuecomment-1185965114)) but I don't think there's a large difference here. I'm ok with this change unless @halter73 has push back.

Reply:
I'm including it on the API review